.gitignore

```diff
@@ -15,6 +15,9 @@
 # ignore vscode settings
 .vscode
 
+# ignore .cargo in CI
+.cargo/
```

Comment:
Does this affect developers when sharing pull requests?

Reply:
It shouldn't since CARGO_HOME on Dev & CI are diffrent paths

Reply:
Where do you see that?

Reply:
undone